src/main/java/com/google/api/codegen/discovery/viewmodel/SampleView.java

```diff
@@ -146,6 +148,17 @@ public String resourceRoot() {
   @Nullable
   public abstract List<String> methodParamAssigments();
 
+  // JS specific section...
+  @Nullable
+  public abstract String paramsVarName();
+
+  /**
+   * Returns true if the page streaming call requires a `params` object to set the `pageToken`
+   * field, even if there are no required params.
```

Comment:
This should be JS specific, I don't think any other language should use it. I've edited the doc string to clarify it's JS only.